packages/forms-web-app/src/pages/process-guide/_translations/en.json

```diff
@@ -0,0 +1,69 @@
+{
+	"applicationSummary": {
+		"heading1": "The process for Nationally Significant Infrastructure Projects (NSIPs)",
+		"paragraph1": "Nationally Significant Infrastructure Projects are submitted to the Planning Inspectorate. We are the government agency who examine the proposal.",
+		"paragraph2": "Anyone can get involved including:",
+		"listItem1": "applicants",
+		"listItem2": "members of the public",
+		"listItem3": "statutory bodies",
+		"listItem4": "charities",
+		"listItem5": "local authorities",
+		"paragraph3": "An applicant submits an application for a development consent order to the Planning Inspectorate. These can be developments like:",
+		"listItem6": "offshore wind farms",
+		"listItem7": "power stations and electric lines",
+		"listItem8": "motorways and other major roads",
+		"listItem9": "railways",
+		"listItem10": "gas pipelines",
+		"paragraph4": "We appoint independent inspectors to examine application and make recommendations to the relevant Secretary of State about whether permission for development should be given.",
+		"paragraph5": "The relevant Secretary of State makes the final decision.",
+		"paragraph6": "There are several stages in the process for considering whether a national infrastructure project should be given consent. The whole process can take around 18 months."
+	},
+	"haveYourSay": {
+		"heading2": "Guide for people or organisations who want to have their say",
+		"paragraph7": "Members of the public can get involved in early consultations before an application has been submitted by contacting the applicant directly.",
+		"paragraph8": "Alternatively you can register to have your say on this website during the pre-examination stage.",
+		"paragraph9": "There is a {{-link}} with information for people or organisations who want to have their say about a national infrastructure project.",
+		"paragraph9LinkText": "guide"
+	},
+	"preApplication": {
+		"subHeading1": "Pre-application",
+		"paragraph10": "This is where the applicant starts to create their application. The applicant is required to consult with people and organisations in the area. They must also create detailed documents about the impact the project could have on the environment.",
+		"paragraph11": "It is important to get involved at this stage to influence the application before the applicant sends it to the Planning Inspectorate.",
+		"linkText1": "Find out what you can do at this stage and check our detailed guides."
+	},
+	"acceptance": {
+		"subHeading2": "Acceptance",
```

Comment:
The naming convention of the keys could be to reset the numbers for each section. This could be named heading1 without conflict as it is nested.